source/_integrations/zha.markdown

```diff
@@ -69,15 +69,17 @@ There is also support for Zigbee grouping of lights, switches, and fans (i.e. su
 
-Before installing the ZHA integration, you will need to buy and connect a Zigbee Coordinator radio adapter or hardware module, those usually come in the form of a USB dongle that plugs directly into the computer that is running the Home Assistant installation. The ZHA integration can work with many different "Zigbee Coordinator" adapters, however, be sure to read the respective sections below about compatible Zigbee radio adapters and hardware modules.
+Once ZHA has been set up and configured with a Zigbee Coordinator radio adapter or module you will be able to directly join/pair any Zigbee Router devices and Zigbee End Devices to the Zigbee network; [regardless of the manufacturer and brand of that Zigbee-based product](#knowing-which-devices-are-supported), (with few [limitations](#limitations)). While backward compatible with devices that use earlier versions of Zigbee Home Automation (ZHA) and Zigbee Light Link (ZLL) protocols, it is generally recommended to buy Zigbee 3.0 (ZB3) compliant devices. Zigbee 3.0 certified devices usually offer better interoperability with more Zigbee gateway solutions out-of-the-box, however [devices that use manufacturer-specific extensions to add non-standard functions and features will still need quirk code in custom device handlers to fully work with ZHA](#how-to-add-support-for-new-and-unsupported-devices).
 
-Once ZHA has been set up and the Zigbee Coordinator radio adapter or module is configured, you will be able to directly join/pair any Zigbee device to the Zigbee network; regardless of the manufacturer and brand of that Zigbee-based product. Note, that while it is generally recommended to buy Zigbee 3.0 compliant devices as those should, in theory, offer greater interoperability, be sure of the sections about which devices are supported and exception handling.
+Because a Zigbee relies on "mesh networking" technology it depends on its Zigbee Router devices for expanding the network coverage and size. These are always mains-powered devices that will act as Zigbee routers (often wrongly referred to as signal repeaters) within the Zigbee network mesh to extend its range and expand the total amount of devices that can be added. Zigbee End Device devices on the other hand will not expand the range of the network as they do not pass along any signals/messages from other devices, these are usually all battery-operated devices, but not always. You should therefore make sure that your Zigbee network has enough Zigbee Router devices, and always begin by adding those before adding Zigbee End devices when getting started.
```

Comment:
There are dedicated signal repeater devices for sale. I don't think the comment is necessary.

```suggestion
Because a Zigbee relies on "mesh networking" technology it depends on its Zigbee Router devices for expanding the network coverage and size. These are always mains-powered devices that will act as Zigbee routers within the Zigbee network mesh to extend its range and expand the total amount of devices that can be added. Zigbee End Device devices on the other hand will not expand the range of the network as they do not pass along any signals/messages from other devices, these are usually all battery-operated devices, but not always. You should therefore make sure that your Zigbee network has enough Zigbee Router devices, and always begin by adding those before adding Zigbee End devices when getting started.
```

Reply:
Changed in https://github.com/home-assistant/home-assistant.io/pull/28869/commits/e9e308f0df0a4dc4eb1b27e706b27d7bfa7b7b1d